icu4c/source/i18n/vtzone.cpp

```diff
@@ -1217,6 +1217,13 @@ VTimeZone::getOffset(UDate date, UBool local, int32_t& rawOffset,
     return tz->getOffset(date, local, rawOffset, dstOffset, status);
 }
 
+void VTimeZone::getOffsetFromLocal(
+    UDate date, UTimeZoneLocalOption nonExistingTimeOpt,
+    UTimeZoneLocalOption duplicatedTimeOpt,
+    int32_t& rawOffset, int32_t& dstOffset, UErrorCode& status) const {
+    tz->getOffsetFromLocal(date, nonExistingTimeOpt, duplicatedTimeOpt, rawOffset, dstOffset, status);
```

Comment:
Add 4 more spaces to indent

Reply:
BTW, this is from your PR. Anyway, I improved readability by adding more indent and matching other methods in the same file.